tests/parsers/test.locale-messagesjson.js

```diff
@@ -92,22 +92,20 @@ describe('LocaleMessagesJSONParser', () => {
     expect(warnings[1].message).toEqual(messages.MISSING_PLACEHOLDER.message);
   });
 
-  it('should be invalid if bad message name', () => {
+  it('should not be an error if bad message name', () => {
     const addonsLinter = new Linter({ _: ['bar'] });
     const localeMessagesJSONParser = new LocaleMessagesJSONParser(`{
   "not-valid": {
     "message": "foo"
   }
 }`, addonsLinter.collector);
     localeMessagesJSONParser.parse();
-    expect(localeMessagesJSONParser.isValid).toEqual(false);
+    expect(localeMessagesJSONParser.isValid).toEqual(true);
     const { errors } = addonsLinter.collector;
-    expect(errors.length).toEqual(1);
-    expect(errors[0].code).toEqual(messages.INVALID_MESSAGE_NAME.code);
-    expect(errors[0].message).toEqual(messages.INVALID_MESSAGE_NAME.message);
+    expect(errors.length).toEqual(0);
   });
 
-  it('should be invalid if bad placeholder name', () => {
+  it('should not be an error if bad placeholder name', () => {
```

Comment:
As I wrote in the first message, I think that we should still raise validation errors for the invalid placeholder names, which would be ignored by Firefox and produce unexpected behaviors at runtime (well, we expect that the invalid placeholder is going to be ignored, but an extension developer that is not aware of this rule will be confused by this behavior and Firefox will not log any warning about it).

Reply:
Ah ha! I think I simply wanted to :fire: too much, not checking the message name but keeping the check for placeholder sounds alright. I'll fix the PR